src/Umbraco.Core/Persistence/Repositories/ILastSyncedRepository.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Umbraco.Cms.Core.Persistence.Repositories;
+
+/// <summary>
+/// Handles saving and pruning of the LastSynced database table.
+/// </summary>
+public interface ILastSyncedRepository
+{
+    /// <summary>
+    /// Used to fetch the last synced internal ID from the database.
+    /// </summary>
+    /// <returns>The Internal ID from the database.</returns>
```

Comment:
```suggestion
    /// <summary>
    /// Fetches the last synced internal ID from the database.
    /// </summary>
    /// <returns>The Internal ID from the database.</returns>
```

It's better to just say what the method does, rather than what it's used for :smile: 

The same goes for the rest of this file and `ILastSyncedManager`